admin-console/src/main/webapp/src/components/EmailTemplates/EmailTemplateTable.tsx

```diff
@@ -0,0 +1,129 @@
+import { Breadcrumb, BreadcrumbItem, Button, PageSection, Title, Toolbar,
+    ToolbarContent, ToolbarItem } from '@patternfly/react-core';
+import { EyeIcon, PencilAltIcon, TrashIcon } from '@patternfly/react-icons';
+import {
+    TableComposable,
+    Tbody,
+    Td,  Th,   Thead,
+    Tr } from '@patternfly/react-table';
+import * as React from 'react';
+import { useParams } from 'react-router';
+import { Link } from 'react-router-dom';
+
+import { useUserPermissions } from '../../app/PermissionContext';
+import { linkTo } from '../../Routes';
+import { useApplicationTypes } from '../../services/EventTypes/GetApplication';
+import { ListEventTypes } from '../EventTypes/ListEventTypes';
+import { ViewTemplateModal } from './ViewTemplateModal';
+
+type ApplicationPageParams = {
+    applicationId: string;
+}
+
+export const EmailTemplateTable: React.FunctionComponent = () => {
+    const { hasPermission } = useUserPermissions();
```

Comment:
To use the `isAdmin`

```suggestion
    const { hasPermission, isAdmin } = useUserPermissions();
```